src/store/spotRequests/index.js

```diff
@@ -0,0 +1,59 @@
+// Import dependencies and constants
```

Comment:
unnecessary comments.
You can remove comments which don't add any value or are pretty obvious.
Please remove your other comments too.